packages/notebook/src/actions.tsx

```diff
@@ -31,6 +31,7 @@ import * as React from 'react';
 import { INotebookModel } from './model';
 
 import { Notebook } from './widget';
+import { JSONObject } from '../../../node_modules/@phosphor/coreutils';
```

Comment:
Here and below: why use the relative path rather than `@phosphor/coreutils`, as used elsewhere?

Reply:
Hmm, that is odd - don't know how that happened (I didn't type it!). Probably a VSCode "Intellisense" things. Latest commit fixes it.